mnefun/_mnefun.py

```diff
@@ -3,7 +3,6 @@
 # Distributed under the (new) BSD License. See LICENSE.txt for more info.
 
 from __future__ import print_function
-
```

Comment:
this was actually intentional to separate `future` imports from standard imports